trunk/SUAVE/Methods/Weights/Correlations/BWB/systems.py

```diff
@@ -75,8 +75,11 @@ def systems(num_seats,  ctrl_type,S_h,S_gross_w, ac_type):
 
     # Furnishings Group Wt
     furnish_wt = ((43.7 - 0.037*min(num_seats,300.))*num_seats + 46.0*num_seats) * Units.lb
-    if ac_type == "business":
-        furnish_wt = S_fus * 11.
+    
```

Comment:
Take it out

Reply:
Take what out?

Reply:
![image](https://user-images.githubusercontent.com/7077520/52031103-bb41e980-24cf-11e9-9603-8ac0b6e286b6.png)

If you're wondering. This is what I see when I look. I want you to remove the commented out code below if it actually exists.